install-extra.sh

```diff
@@ -0,0 +1,56 @@
+###########################################################################################
+#                                                                                         #
+# Note: if this script is updated make sure to update the copy in xsoft's home directory! #
+#                                                                                         #
+###########################################################################################
+
```

Comment:
```suggestion

set -euxo pipefail

```

For this kind of thing failing fast seems like the best option. There are a few places where something could fail but the script might carry on and do unintended things, like if the module command fails.

Reply:
Good point, added (and tested) in 12f2ea9.